pkg/get/get.go

```diff
@@ -284,3 +289,73 @@ func GetBinaryName(tool *Tool, os, arch, version string) (string, error) {
 
 	return "", errors.New("BinaryTemplate is not set")
 }
+
+// DownloadList verifies tool exists and returns a download list
+func DownloadList(tools Tools, toolArgs []string, version string) (Tools, error) {
+	arkadeTools := []Tool{}
+outer:
+	for _, arg := range toolArgs {
+		name := arg
+
+		// Handle version specified tool name
+		if i := strings.LastIndex(arg, "@"); i > -1 {
+			if len(version) > 0 {
+				return nil, fmt.Errorf("cannot specify --version flag and @ syntax at the same time")
```

Comment:
Perhaps this error should include the string in question i.e. `arg` in this case, so that the user knows which tool caused the error when downloading a longer list it will be more specific and actionable.